### ISSUE TITLE
Add list view option to home tab.

### ISSUE THREAD

**Issue by pganssle** [vector-im/riot-android#1408]
I dislike the new "Home/Favorites/People/Rooms" view, mostly because the "Home" view seems to be enforcing icon-view only. I would *much* rather have one vertical scroll with folders than several horizontal scrolls with icons.

I think you should be able to switch the home view to use the old text view rather than the icon view. Perhaps this could be a tab-specific UI option, so that any of the 4 tabs can be toggled between icon and list view.

**Comment by benqrn** [comment]
I agree, the older style was easier to navigate, especially with one hand.  There could be an option to toggle between them, and also the icon-view should have a line wrap option as well instead of having to side-scroll.

**Comment by smichel17** [comment]
What if you could choose the default tab to view on startup? I like the current home screen for how many rooms it can show (a list of equivalent length is harder to navigate), but most of the time I only care about my favorites. So,  I don't want any new functionality; I just don't want to have to side-scroll to get to the favorites list.
